<h1 id="bio490s---methods-in-computational-biology-genomics">BIO490S - Methods in Computational Biology &amp; Genomics</h1>
<p>Instructor: C. Ryan Campbell Email: <a href="c.ryan.campbell@duke.edu" class="uri">c.ryan.campbell@duke.edu</a> Office: BioSci 311 Hours: Mon 3-5, Wed 10-12, or by appointment</p>
<h2 id="learning-objectives">Learning Objectives</h2>
<ol type="1">
<li>teaching programming basics within a biology framework (i.e. basic building blocks for students to use biological software):
<ul>
<li>using command line interfaces</li>
<li>accessing supercomputers/clusters</li>
<li>building scripts</li>
<li>using biological programs (bwa, samtools, etc)</li>
</ul></li>
<li><p>the statistics that underlie these tools, the whole suite of measures used in &quot;genomics&quot; and probably not covered in Bio 204.</p></li>
<li><p>combine #1 &amp; #2 to test one of a range of biological questions, answerable with RNAseq data, such as: what is the effect of hibernation (lemur), morph type (cricket), tissue type (crayfish), closing mechanism (venus flytrap), and salty ecotypes (cotton) on gene expression across the tree of life.</p></li>
</ol>
<h3 id="prerequisites">Prerequisites</h3>
<ul>
<li>One of BIO201/202</li>
<li>STA101 or higher OR BIO204</li>
<li>Some coding background (computer language independent)</li>
</ul>
<h3 id="schedule">Schedule</h3>
<table>
<colgroup>
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 56%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th>Day</th>
<th>Date</th>
<th>Topic</th>
<th>Due</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Aug 29</td>
<td>Welcome/Syllabus/&quot;What is 'Genomics'?&quot;</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Aug 31_</td>
<td><em>Setup - Terminal Basics</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Sep 5</td>
<td>How We Got Here - History of &quot;Genomics&quot;</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td><em>Sep 7</em></td>
<td><em>Intro to git</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Sep 12</td>
<td>Genomic Methods (Long v. Short Reads)</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Sep 14_</td>
<td><em>Intro to bin/bash language</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Sep 19</td>
<td>SNPs v. Genomes</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Sep 21_</td>
<td><em>Writing a Script, Loops/Boolean</em></td>
<td><em>Project Choice</em></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Sep 26</td>
<td>Misc-omes: Transcriptomes/Epigenomes/etc</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Sep 28_</td>
<td><em>Regular Expressions - grep</em></td>
<td><em>HW: Regular Expression/grep</em></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Oct 3</td>
<td>Statistical Methods (pt I) - BLAST, BUSCO, measures of similarity</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td><em>Oct 5</em></td>
<td><em>Stat Methods pt 1</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Oct 10</td>
<td>Statistical Methods (pt II) - Permutation Tests, experimental stats</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Oct 12_</td>
<td><em>Stat Methods pt 2</em></td>
<td><em>Project Sketch (Methods)</em></td>
</tr>
<tr class="odd">
<td>Tu - No Class</td>
<td>Oct 17</td>
<td>Fall Break -- No Class</td>
<td></td>
</tr>
<tr class="even">
<td>Th - Lecture</td>
<td>Oct 19</td>
<td>Future Genomic Tech - TenX, long reads, Oxford Nano</td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Oct 24</td>
<td>Statistal Methods (pt III) - Bayesian Stats, Simulations</td>
<td>HW: Statistical Methods</td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Oct 26_</td>
<td><em>Stat Methods pt 3</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Oct 31</td>
<td>What can we learn from genomes? (Pop Gen, PSMC)</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td><em>Nov 2</em></td>
<td><em>Submitting to the Cluster</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Nov 7</td>
<td>Admixture/PopGen from Genomes</td>
<td>HW: Paper Assessment</td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td><em>Nov 9</em></td>
<td><em>Downloading Public Data - SRA</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Nov 14</td>
<td>Genomic &quot;Islands&quot; of Speciation</td>
<td>Project Rough Draft 1</td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Nov 16_</td>
<td><em>Using Software - Readme.txt and Flags!</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Nov 21</td>
<td>New Directions in Genomics: Introns &amp; Alternative Splicing</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Nov 23_</td>
<td><em>THANKSGIVING -- No Class Git repositories and version tracking</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Nov 28</td>
<td>Non-Model Genomics - Possibilities and Pitfalls</td>
<td></td>
</tr>
<tr class="even">
<td><em>Th - Lab</em></td>
<td>_Nov 30_</td>
<td><em>Project Presentations/Student Choice Lecture</em></td>
<td></td>
</tr>
<tr class="odd">
<td>Tu - Lecture</td>
<td>Dec 6</td>
<td>Project Presentations</td>
<td>Final Project (Present)</td>
</tr>
<tr class="even">
<td>Th - Lecture</td>
<td>Dec 8</td>
<td>Project Presentations</td>
<td>Final Project (Present)</td>
</tr>
<tr class="odd">
<td>Exam</td>
<td></td>
<td>Final Exam - In Class Activity/Mini-Project</td>
<td></td>
</tr>
</tbody>
</table>
